Adding section Promise

Agnehska opened the pull request:
1. Поправила верстку первой секции (разделила компоненты и стили по папочкам)
2. Сверстала 2 секцию

Agnehska committed:
Adding section Promise

AndreiMedvedevSaratov commented on src/components/Promise/Promise.jsx:
Предлагаю декомпозировать на 3 компонента - основание для компонента - это колонка.
В неё передаём массив карточек.
И нужно будет отрисовывать основываясь на массиве карточек.

AndreiMedvedevSaratov commented on src/components/Promise/Promise.jsx:
```diff
@@ -0,0 +1,77 @@
+            </h2>
+            <div className="promise__row">
+                <div className="promise__column">
+                        <Promise__cart 
```
Комментарий к конкретной строчке

Agnehska committed:
Adding sections Popular and IdealHome

1. Добавила секцию Popular и сделала фильтр.
2. Добавила секцию IdealHome и настроила табуляцию.

Agnehska committed:
Решение homework2

Добавила решение задач
